make/autoconf/flags-ldflags.m4

```diff
@@ -77,7 +77,8 @@ AC_DEFUN([FLAGS_SETUP_LDFLAGS_HELPER],
         -fPIC"
 
   elif test "x$TOOLCHAIN_TYPE" = xxlc; then
-    BASIC_LDFLAGS="-b64 -brtl -bnorwexec -bnolibpath -bexpall -bernotok -btextpsize:64K \
+    # -bexpall should be used instead of -bnoexpall if -qvisibility option is not defined
```

Comment:
If you hard-code for the assumption that xlc v16 is being used, then this comment does not make any sense..?

Reply:
Thanks, @magicus .
I removed the comment.